### PR TITLE
Update

### PR DESCRIPTION
**Pull request by cedricngakou**
@egbendito  i set use_case to NA because i did not identified the use case name related to this data on the use case list. Please take a look. Thanks

### CODE REVIEW

File: scripts/eia/doi_Nigeria-ATAFI-Validations.R [hunk 1]
```diff
@@ -18,20 +18,17 @@ carob_script <- function(path) {
...
       authors = "Mary Jane;John Do",
-      data_citation="...",
-      data_institute = NA ,
+      data_institute = "AAA-BBB" , # partner name 
       title = NA,
       description = "Nigeria-ATAFI-Validations",
       group = group,
       publication=NA,
-      license = 'Some license here...',
+      use_case = NA,
```
Comment (egbendito) [inline scripts/eia/doi_Nigeria-ATAFI-Validations.R:27]: The use case is `WA-Rice-ATAFI/MOVE` on [terms/values/values_eia_usecases.csv#L8](https://github.com/EiA2030/eia-carob/blob/6583263029293c52ba7d574b2536c1a31c9db70d/terms/values/values_eia_usecases.csv#L8)

File: scripts/eia/doi_Nigeria-ATAFI-Validations.R [hunk 2]
```diff
@@ -18,20 +18,17 @@ carob_script <- function(path) {
    
    meta <- data.frame(
       # Need to fill-in metadata...
-      # carobiner::read_metadata(uri, path, group, major=2, minor=0),
-      uri = carobiner::simple_uri(uri),
```
Comment (egbendito) [inline scripts/eia/doi_Nigeria-ATAFI-Validations.R:22]: We need `uri`

File: scripts/eia/doi_Nigeria-ATAFI-Validations.R [hunk 3]
```diff
@@ -18,20 +18,17 @@ carob_script <- function(path) {
...
-      dataset_id = uri,
```
Comment (egbendito) [inline scripts/eia/doi_Nigeria-ATAFI-Validations.R:23]: We need `dataset_id`

File: scripts/eia/doi_Nigeria-ATAFI-Validations.R [hunk 4]
```diff
@@ -18,20 +18,17 @@ carob_script <- function(path) {
...
       authors = "Mary Jane;John Do",
-      data_citation="...",
-      data_institute = NA ,
```
Comment (egbendito) [inline scripts/eia/doi_Nigeria-ATAFI-Validations.R:26]: This is `AfricaRice` from [values/values_institute.csv#L3](https://github.com/reagro/terminag/blob/483e21820604113ae7adb8a9a190ab4fcd125382/values/values_institute.csv#L3)

**Review by egbendito — CHANGES_REQUESTED** [review]

**Comment by cedricngakou** [comment]
Thanks, I just reviewed it now, you can check, approve and merge .

**Review by egbendito — APPROVED** [review]

**Comment by egbendito** [comment]
Great, thanks!

**Pull request merged**